presto-sheets/src/test/resources/gsheets-creds-base64-encoded.props

```diff
@@ -0,0 +1 @@
c3RvdGVzdCIsCiAgInByaXZhdGVfa2V5X2lkIjogIjIxMDFiMzBkNjM2ZWU3ZWNkZmFjN2U3M2JiOThmOTVjNjY5OTE5ZmQiLAogICJwcml2YXRlX2tleSI6ICItLS0tLUJFR0lOIFBSSVZBVEUgS0VZLS0tLS1cbk1JSUV1d0lCQURBTkJna3Foa2lHOXcwQkFRRUZBQVNDQktVd2dnU2hBZ0VBQW9JQkFRQ3RjdGl3YmtpZ0s0N29cbjR3VGFJU0lNcEFaTXV6V0h4emF4andReUYzZk93QUZSWlk0Vm8yOTlkWi9zeDcvdU94Nk5EMmRIbDFZa2JZdGZcbmlRWENxTzFQZTJkeklrRjB3ZHh2OXE0bFBON0ZHbnY2NGYvcWlwSElRaHRJbGUvcnRORmJGNlQ4WVV6T08vZ0Vcbmg0THBiYnN0dHNYb1FRRDh6bnV0Y3l1WDE3SmlXRHFaQUkyM2VrOW9Qa2ZmbkdIOVVVSDQzZmh3RWFuZFZIb3ZcbjB2dEpyZUd3TkxpeWlkS2xHVUZJU0NPVlQvSmJaRzJoN0dsVm90SUV6Z0QvRXQvbERQNXYrM1IxZ0JmQkVYa2RcbkxHWlA5V2poRFFkcWlVYnZmYnNvYVFxSDRiRWtlQzFxQ0dheDFxVmk3L0V6ajYzQjhEajdEWFdzTDVlTzRaUGNcbkc5WUt2aDJaQWdNQkFBRUNnZjhhYTZiVndON0ZYOHUwNzRZZnZMNWNEL0VuWEpPeHQ0VFpkOXoySkt4VEcxZDhcbnM5VTM5NjVwc3VTM3Q2UXFCekVWT2lnWE1GZW51bGorYkVSc3RJZWRVMHh5Rk0xbS9MbnQwT1FFenFqTlJWSjZcblhDQ2NPbEZ3dUlibGtYc3dBanRpMkQ3WTBHK2Y2MUJJbjh0VWZWOUdWdFBEeFBESFQvVGFGWlcyMXZ2dmZiMlFcbkI3YWhUb0hrRG9hR01yRXdveXlZM1plZkFzMjZ3NGlNY216WXIvTndrTGk5bkZMU1pxVFgvaTBmNy84eDdTUWJcbkpudUFMdWtHc2VybDZlUW5tOCs4czg1bTZ2Njk3MVNaTWlqR2VudFdzVko5NWNDd0prZXZoSHhWcVU4MHMrdHRcblJNajRZZ0tIUU9lOWJNTGhmTUJYcis4bldCRlcyRkNVakxmVDRBa0NnWUVBMTdCSFg3V3RHdzZ3OEdlYWxRZ1lcbnFIeFJYZlhNOG5yZTN2S20rVTJKREVBNHlrT2kzdi9IQ3lySmt4Z0dhS01HQkVmZFJRVmpFZ1pJM0JmbEg4aWFcbnpzcHNJZy9laHFIcWNsYTUxYXowVXF6cG5WQTlPQlVDcXVrK3AzTW9ZcXQ5cWtOZkUxSFZ1d2FpS0ZBcjBqUW1cbjFJUXorUjkyMmZMOTZ2VWJRbFMrSGZVQ2dZRUF6ZDJUbVF6ZEcxR2N5YnpSS2Y1T3diRkJZU29qdmFrL2NpSzdcbm43ZjJNTFVkMjBGRGoxOUE1dE5DMkVSUms2M3VndFpkYkVyZWVyTHNRQkR3TGJGNVZtdkp6OWlXTGNTaFp6U09cbndHZ09BL1psQzJuSytOdWdxMzExTGNQL3JTak5Ua0lnQ1pDNkNnNXZFcS9uSGtEaVIzcUIwbnplSzhpUWdSczdcbmYwNlJOcFVDZ1lCTFdLczJSaEw0dHNNYzJEUnFCM0FGdTVmNUNOclZnaExGTTJNZEIvYnVMcWtvRDBTZ1BoM2NcblN0QXI4R0tjdndabERHM0JwTVlkRDlENEFlanFsM3RzN2Mya2dJS3Q2WFlyQnh6WUJoazdvR2ZqcXBvWU0wN0JcblFjVy9YVDBvYXR2MVZKK3I4aFNGeHlWeWk3OGpLVkMvUUllZmFWeHRlck0xRnZhMHowR3kyUUtCZ1FDL0x2ckdcbjFWSjE3M29iUkRZL3VJUHRqNjgzREVwNjliWU5Ta0pTbHlhVW4yOGwwdHRxUjd6ZllPMGdLWHBWR3Q5NU5KazRcblcrTjNzYmlEMnZjaWlFSTJSdzM1NzV1d3ozNVJlVmJsU3YxUklyOCtOVGFobkYzL1JSalFzc2c4U0xzNTVXRVpcbllRWFZGd2lDSTBnUktaSzdiUU1wdG5zaDBEYWJIKzVHVVhHOGJRS0JnSFRPUkxaeDlTcG1TaW90UFdCVFpBMEtcbllTaGU2TGwxM0xBbVgyc3NmTUp2TGZuVkhHdnJ6SlpXdnVpUUZhYnJqYWFja0lLUkxPRlErNitNdUluNUR5TVNcbnI4S0taeW9HeHBMZ1JXbW9wRHdkWGRtdHZxa1VmbXZwU1JjYUJ2a2hUTHB5MnFuZWFpRlYrT0loRTdEYk9lUk1cblVjUjQ3Mmd0SlkwZDRLVG1hdWZLXG4tLS0tLUVORCBQUklWQVRFIEtFWS0tLS0tXG4iLAogICJjbGllbnRfZW1haWwiOiAicHJlc3RvdGVzdEBwcmVzdG90ZXN0LmlhbS5nc2VydmljZWFjY291bnQuY29tIiwKICAiY2xpZW50X2lkIjogIjExNjk4NTIyMjg1MjE4NDM3NjI5MyIsCiAgImF1dGhfdXJpIjogImh0dHBzOi8vYWNjb3VudHMuZ29vZ2xlLmNvbS9vL29hdXRoMi9hdXRoIiwKICAidG9rZW5fdXJpIjogImh0dHBzOi8vb2F1dGgyLmdvb2dsZWFwaXMuY29tL3Rva2VuIiwKICAiYXV0aF9wcm92aWRlcl94NTA5X2NlcnRfdXJsIjogImh0dHBzOi8vd3d3Lmdvb2dsZWFwaXMuY29tL29hdXRoMi92MS9jZXJ0cyIsCiAgImNsaWVudF94NTA5X2NlcnRfdXJsIjogImh0dHBzOi8vd3d3Lmdvb2dsZWFwaXMuY29tL3JvYm90L3YxL21ldGFkYXRhL3g1MDkvcHJlc3RvdGVzdCU0MHByZXN0b3Rlc3QuaWFtLmdzZXJ2aWNlYWNjb3VudC5jb20iCn0K
```

Comment:
are there any quota on this? If we start to use travis on for each PR we might hit some google limits here.

Reply:
https://github.com/prestosql/presto/pull/973 allows us to run certain tests on master, but not on PR.

btw what are these credentials to? What account?

Reply:
Api docs says unlimited quota, but there should be a limit of 300 reqs per sec.
I created a test google account - `prestosqltest@gmail.com`. Please let me know how I can share the login info etc, or if there is another test account.

Reply:
> I created a test google account - `prestosqltest@gmail.com`. Please let me know how I can share the login info etc, or if there is another test account.

cc @martint @electrum

Reply:
We go with this account, and when it is checked in, we can switch to an account at prestosql.io, which is in gsuite.